When configured with `--disable-xft` fvwm3 fails to build.

## Upfront Information

* Fvwm3 version (run: `fvwm3 --version`) 

1.0.4 / master

* Linux distribution or BSD name/version 

Gentoo

* Platform (run: `uname -sp`)

Linux monolith.home.footclan.ninja 5.18.1-gentoo-r2 #1 SMP PREEMPT_DYNAMIC Sat Jun 4 01:25:23 AEST 2022 x86_64 AMD Ryzen 9 5950X 16-Core Processor AuthenticAMD GNU/Linux

## Expected Behaviour

What were you trying to do?  

Build failures were reported through automation, tracked down to gentoo `USE=truetype` not being enabled for certain profiles / users.

I was able to replicate this issue and track it down to `./configure ... --disable-xft`, which I am also able to replicate on master.

With this option set, the following error is received and the build fails.

```
FftInterface.h:23:9: error: unknown type name 'XftFont'
```

[build-failure-git-master.txt](https://github.com/fvwmorg/fvwm3/files/8837788/build-failure-git-master.txt)

## Actual Behaviour

What should have happened, but didn't?

The software should build!

## Steps to Reproduce

```bash
git clone https://github.com/fvwmorg/fvwm3.git
cd fvwm3
/autogen.sh && ./configure --disable-xft && make -j
```

--- a/configure.ac
+++ b/configure.ac
@@ -537,28 +537,11 @@ fi
 AC_SUBST(Xcursor_CFLAGS)
 AC_SUBST(Xcursor_LIBS)
 
-# ********* xft
-problem_xft=""
-
-AC_ARG_ENABLE(xft,
-  AS_HELP_STRING([--disable-xft],[disable Xft anti-aliased font rendering]),
-  [ if test x"$enableval" = xyes; then
-    with_xft="yes, check"
-  else
-    with_xft="no"
-    problem_xft=": Explicitly disabled"
-  fi ],
-  [
-    with_xft="not specified, check"
-  ]
-)
-
 freetype_CFLAGS=""
 freetype_LIBS=""
 AH_TEMPLATE([HAVE_XFT],[Define if Xft library is used.])
 AH_TEMPLATE([HAVE_XFT2],[Define if Xft 2 library is used.])
 AH_TEMPLATE([HAVE_XFT_UTF8],[Define if Xft library can handle utf8 encoding])
-if test ! x"$with_xft" = xno; then
   AC_MSG_CHECKING([whether pkg-config could find freetype2])
   if "${PKG_CONFIG}" --exists freetype2; then
     AC_MSG_RESULT([yes])
@@ -641,7 +624,6 @@ if test ! x"$with_xft" = xno; then
     fi
   fi
 
-fi
 AC_SUBST(Xft_LIBS)
 AC_SUBST(Xft_CFLAGS)
 

